Apps/Src/UpdateVelocity.c

```diff
@@ -0,0 +1,36 @@
+#include "SendTritium.h"
+#include "CarState.h"
+#include "UpdateVelocity.h"
+
+// As of 03/13/2021 the function just returns a 1 to 1 conversion
+static convertPedaltoMotorPercent(uint8_t pedalPercentage){
+    return pedalPercentage;
+}
+
+void Task_UpdateVelocity(void* p_arg){
+    
+    car_state_t *car_state = (car_state_t *) p_arg;
+
+    OS_ERR err;
+
+    //Set desired velocity to 0
+    car_state->DesiredVelocity = 0;
+
+    while(1){
+        //Read global pedal percentage
+        uint8_t pedalPercentage = car_state->AccelPedalPercent;
+
+        if(car_state->CruiseControlEnable){
+            if(car_state->CruiseControlSet){
+                car_state->DesiredVelocity = car_state->CruiseControlVelocity;
```

Comment:
You should also set the desired motor current here. For now it's probably best to set it to 100% (1.0f) but that might be a parameter we want to tweak in the future.